docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+   "metadata": {},
+   "source": [
+    "## Run Molecular Dynamics simulation\n",
+    "\n",
+    "We run the MD at 300 K with an NVT Berendsen thermostat. \n",
+    "\n",
+    " TODO: how do we manage remote execution examples code-testing-wise?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Creating job md_chunk_0\n",
+      "ExPyRe md_chunk_0 constructor done stage in files 1675187004.1068766\n",
+      "Starting job for md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6\n",
+      "Gathering results for md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6 remote 205293\n",
+      "Waiting for job md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6 up to 3600 s: \n",
+      "qqrr\n",
+      "Running wfl.generate.md._sample_autopara_wrappable with num_python_subprocesses=4, num_inputs_per_python_subprocess=1\n"
+     ]
+    }
+   ],
+   "source": [
+    "outputs = OutputSpec(\"2.ch.rdkit.md.traj.xyz\")\n",
+    "\n",
+    "md_params = {\n",
+    "    \"steps\": 80,\n",
+    "    \"dt\": 0.5,  # fs\n",
+    "    \"temperature\": 300,  # K\n",
+    "    \"temperature_tau\": 500,  \n",
+    "    \"results_prefix\": \"xtb_\",\n",
+    "    \"traj_step_interval\": 5}\n",
+    "\n",
+    "remote_info = {\n",
+    "    \"sys_name\" : \"local\", \n",
+    "    \"job_name\" : \"md\", \n",
+    "    \"resources\" : { \n",
+    "        \"max_time\" : \"15m\",\n",
+    "        \"num_cores\" : 4,\n",
+    "        \"partitions\" : \"any$\"}, \n",
+    "    \"partial_node\": True,\n",
+    "    \"check_interval\": 5,\n",
+    "    \"num_inputs_per_queued_job\" :20,\n",
+    "    \"pre_cmds\" :[\"conda activate dev\"]\n",
+    "}\n",
+    "\n",
+    "# set to None for github testing purposes\n",
+    "remote_info=None\n",
+    "\n",
+    "inputs = md.sample(\n",
+    "    inputs=inputs, \n",
+    "    outputs=outputs,\n",
+    "    calculator=xtb_calc,\n",
+    "    autopara_info = AutoparaInfo(\n",
+    "        remote_info=remote_info),\n",
+    "    **md_params\n",
+    "    )\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filter out unreasonable structures\n",
+    "\n",
+    "While diverse training set leads to better model extrapolation, structures too dissimilar to the region of interest are fitted at the expense of accuracy elsewhere. One way to spot structures somewhat distant from equilibrium is by checking for high force components. Below we exclude such structures via one of wfl's filtering functions. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "def are_xtb_force_comps_reasonable(at):\n",
+    "    force_comps = at.arrays[\"xtb_forces\"]\n",
+    "    return np.all(np.abs(force_comps) < 8 )\n",
```

Comment:
quite minor, but isn't it more physically reasonable to consider the total force on each atom, so `np.all(np.linalg.norm(force_comps, axis=1) < 8)` ?